js/gulp/package-task.js

```diff
@@ -51,7 +51,7 @@ const createMainPackageJson = (target, format) => (orig) => ({
     module: `${mainExport}.dom.mjs`,
     types: `${mainExport}.node.d.ts`,
     unpkg: `${mainExport}.es5.min.js`,
-    jsdelivr: `${mainExport}.es5.min.js`,
+    jsdeliver: `${mainExport}.es5.min.js`,
```

Comment:
I don't think this is a typo

https://www.jsdelivr.com/

Reply:
Thank you for pointing it out. I have just reverted the changes.